hackernews.el

```diff
@@ -490,6 +491,11 @@ which see."
   (interactive)
   (hackernews--visit (point) hackernews-internal-browser-function))
 
+(defun hackernews-button-mark-as-visited ()
+  "Mark button under point as visited."
+  (interactive)
+  (hackernews--visit (point) 'ignore))
```

Comment:
`ignore` is a function symbol, so it can/should be quoted as `#'ignore`. It shouldn't make a difference in this case, but it's better to be consistent with quoting. A small advantage is that the byte-compiler will complain if we ever mistype the function symbol's name, e.g. while refactoring.